checkers/baseapp/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
```

Comment:
what is the directory baseapp/? it looks like this code should be in mysite/checkers/modelrs.py